app/src/main/java/com/sun/imageeditor/data/model/Photo.kt

```diff
@@ -1,3 +1,3 @@
 package com.sun.imageeditor.data.model
 
-data class Photo(id: String)
+data class Photo(val id: String, val title: String, val photoUrl: String)
```

Comment:
Em viết theo dạng: mỗi thuộc tính đều xuống một dòng riêng -> dễ đọc, check bug
vl: Photo(
       val id: String,
       val title: String,..
       ...)

Reply:
E sửa trên toàn bộ các data class nhé